sycl/include/sycl/vector_preview.hpp

```diff
@@ -335,6 +335,26 @@ __SYCL_DEFINE_BF16_VECSTORAGE(4)
 __SYCL_DEFINE_BF16_VECSTORAGE(8)
 __SYCL_DEFINE_BF16_VECSTORAGE(16)
 #undef __SYCL_DEFINE_BF16_VECSTORAGE
+
+// FIXME: Remove this class after eliminating setValue() and getValue()
+// dependencies from math operations on sycl::vec.
+// This class is a friend of sycl::vec and exposes getValue/setValue
+// that are used by sycl::vec math operations.
+template <typename VecT> class VecGetterSetter {
```

Comment:
Nit; I would probably name it `VecAccess` or something like that instead, so we can extend it with other private/protected members in the future, in case we need it.

Reply:
Sure.